### PR TITLE
Add Sanrio characters Picross

### PR DESCRIPTION
**Pull request by yoshiftj**
Adds Sanrio characters Picross.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/sanrio-characters-picross/game.dat [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title = "Sanrio characters Picross"
+description = "This is a puzzle game using Sanrio characters.\n\nSolve Picross puzzles to unveil pictures and get cute stickers to decorate your screen with!\n\nOr use them on game backgrounds!\n\nEnjoy Picross with your own background!"
```
Comment (CodingKoopa) [inline games/sanrio-characters-picross/game.dat]: Remove the `\n` characters.

Comment (CodingKoopa) [inline games/sanrio-characters-picross/game.dat]: This should be phrased like "Sanrio characters Picross is a puzzle game where you solve..."

Comment (CodingKoopa) [inline games/sanrio-characters-picross/game.dat]: > get cute stickers to decorate your screen with!\n\nOr use them on game backgrounds!\n\nEnjoy Picross with your own background!

This should be written more neutrally, without the exclamation marks.

File: games/sanrio-characters-picross/game.dat [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+needs_system_files = false
+needs_shared_font = false
+game_type = "eshop"
+
+[[releases]]
+title = "00040000001D3100"
+region = "eur"
```
Comment (CodingKoopa) [inline games/sanrio-characters-picross/game.dat]: Add a USA release if there is one.

**Comment by yoshiftj** [comment]
Isn't it 3 screenshots if the game performs poorly?

File: games/sanrio-characters-picross/game.dat [hunk 3]
```diff
@@ -0,0 +1,10 @@
+title = "Sanrio characters Picross"
+description = "Sanrio characters Picross is a puzzle game using Sanrio characters. Solve Picross puzzles to unveil pictures and get cute stickers to decorate your screen with, or use them on game backgrounds. Enjoy Picross with your own background.
```
Comment (Leo626) [inline games/sanrio-characters-picross/game.dat]: You are missing the `"` at the end of the sentence.

**Comment by Leo626** [comment]
@yoshiftj Only if the game cannot progress past the title screen, then 1 screenshot could be allowed. You're showing gameplay which means the game can progress past the title screen.

**Comment by yoshiftj** [comment]
How do I add my additional screenshots?

**Comment by Leo626** [comment]
The file names of the screenshots should be descriptive like `title-screen.png`.

Also add `usa` and `jpn` releases. Title IDs are `00040000001D3400` and `00040000001D2A00`.

File: games/sanrio-characters-picross/game.dat [hunk 4]
```diff
@@ -0,0 +1,22 @@
+title = "Sanrio characters Picross"
+description = "Sanrio characters Picross is a puzzle game using Sanrio characters. Solve Picross puzzles to unveil pictures and get cute stickers to decorate your screen with, or use them on game backgrounds. Enjoy Picross with your own background."
```
Comment (CodingKoopa) [inline games/sanrio-characters-picross/game.dat]: > Solve Picross puzzles

This should be rephrased to "The player solves Picross puzzles..."

Comment (CodingKoopa) [inline games/sanrio-characters-picross/game.dat]: > Enjoy Picross with your own background

This still isn't neutral enough. Considering the description already says "or use them on game backgrounds", this sentence can probably be removed altogether.

File: games/sanrio-characters-picross/game.dat (hunk 2 above)
Comment (CodingKoopa) [reply games/sanrio-characters-picross/game.dat]: I'll add that the USA release needs to be first as well.

File: games/sanrio-characters-picross/game.dat [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+needs_system_files = false
+needs_shared_font = false
+game_type = "eshop"
+
+[[releases]]
+title = "00040000001D3100"
+region = "eur"
+release_date = "2018-06-28"
+
+title = "00040000001D3400"
```
Comment (CodingKoopa) [inline games/sanrio-characters-picross/game.dat]: For each new release, you need to have `[[releases]]` again.

File: games/sanrio-characters-picross/game.dat [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
```
Comment (CodingKoopa) [inline games/sanrio-characters-picross/game.dat]: This newline is unneeded.

File: games/sanrio-characters-picross/game.dat [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+region = "usa"
+
```
Comment (CodingKoopa) [inline games/sanrio-characters-picross/game.dat]: Ditto.

File: games/sanrio-characters-picross/game.dat [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+release_date = "2018-07-19"
+
+title = "00040000001D2A00"
```
Comment (CodingKoopa) [inline games/sanrio-characters-picross/game.dat]: `[[releases]]`.

File: games/sanrio-characters-picross/game.dat [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+
```
Comment (CodingKoopa) [inline games/sanrio-characters-picross/game.dat]: This newline is unneeded.

File: games/sanrio-characters-picross/game.dat [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+region = "jpn"
+
```
Comment (CodingKoopa) [inline games/sanrio-characters-picross/game.dat]: Ditto.

File: games/sanrio-characters-picross/game.dat (hunk 6 above)
Comment (yoshiftj) [reply games/sanrio-characters-picross/game.dat]: Formatting on mobile. This site is a nightmare on mobile. I'll format when at a desktop.

File: games/sanrio-characters-picross/game.dat (hunk 4 above)
Comment (yoshiftj) [reply games/sanrio-characters-picross/game.dat]: This is pulled directly from the Nintendo's website description of the game. Will remove all additional text past the first paragraph.

**Comment by CodingKoopa** [comment]
Closing due to inactivity.

**Pull request closed without merging**